src/components/AssetsTable/AssetsTable.test.jsx

```diff
@@ -81,9 +81,21 @@ const numberOfImages = defaultProps.assetsList.filter(asset => asset.thumbnail).
 const numberOfStudioButtons = defaultProps.assetsList.filter(asset => asset.portable_url).length;
 const numberOfWebButtons = defaultProps.assetsList.filter(asset => asset.external_url).length;
 
+let parentElement;
+let querySelector;
 let wrapper;
 
 describe('<AssetsTable />', () => {
+  beforeEach(() => {
+    querySelector = global.document.querySelector;
```

Comment:
Why do you need a reference to the querySelector method? I see you're changing its implementation below, right? Does this reference get used in some way?

Reply:
I save a reference so I can return the function back to normal in the `afterEach` function with `document.querySelector = querySelector`.

Reply:
Oh, I see. I missed that. Thank you for explaining.